src/emu/render.cpp

```diff
@@ -1585,6 +1585,15 @@ void render_target::load_layout_files(const internal_layout *layoutfile, bool si
 	if (singlefile)
 		return;
 
+	// if override_artwork defined, load that and return
+	if (m_manager.machine().options().override_artwork())
+	{
+		if (load_layout_file(m_manager.machine().options().override_artwork(), m_manager.machine().options().override_artwork()))
+			return;
+		if (load_layout_file(m_manager.machine().options().override_artwork(), "default"))
+			return;
+	}
+
```

Comment:
You can't return early here - you still need to hit the block below where it checks for systems with three or more screens and generates additional views.  Without this, bad things happen when you add additional video cards to computers or enable debug terminals on arcade hardware.  You'll have to skip over the rest of the layout search if this succeeds but still generate views.

Reply:
Ok, that's easily fixed. :)